src/main/resources/system.properties

```diff
@@ -1,4 +1,4 @@
 
-basicURL=http://forge.f-d.com.ua
+basicURL=http://forge.f-d.com.ua/auth/login
```

Comment:
props should contain only http://forge.f-d.com.ua. otherwise it would be hard to use it anywhere, you'll have to cut it all the time. parts such as /auth/login should be added elsewhere, for example in PageFactory